test/parser/plyqlParser.mocha.js

```diff
@@ -967,6 +967,24 @@ describe("SQL parser", () => {
       expect(parse.expression.toJS()).to.deep.equal(ex2.toJS());
     });
 
+    it("should work with SELECT character set introducer", () => {
```

Comment:
you can make this test a lot smaller. all you are testing here is the string literal thing